### ISSUE TITLE
Add function to enable FireBase Logging

### ISSUE THREAD

**Issue by Bojhan** [prescottprue/redux-firebasev3#32]
It would be great if we have a function to enable FireBase Logging. Its mighty useful and now I have to add a line to compose.js every new release.

**Comment by prescottprue** [comment]
@Bojhan Nice suggestion!

Just to confirm, you are talking about enabling database logging through `database.enableLogging()` method right? Like so:

``` js
firebase.database.enableLogging(true);
```

If this is what you mean, it should be easy enough. So it would be enabled through config like this:

``` js
// equivalent to firebase.database.enableLogging(true);
const createStoreWithFirebase = compose(
  reduxFirebase(config, { enableLogging: true }),
)(createStore)
```

It looks like you can also pass a function to enable logging like this:

``` js
firebase.database.enableLogging((logMessage) => {
  // Add a timestamp to the messages
  console.log(new Date().toISOString() + ': ' + logMessage);
})
```

I like the idea of keeping that functionality, so it would end up looking like this:

``` js
const createStoreWithFirebase = compose(
  reduxFirebase(config, {
    enableLogging: (logMessage) => {
      // Add a timestamp to the messages
      console.log(new Date().toISOString() + ': ' + logMessage);
    }
  }
),
)(createStore)
```

What are your thoughts?

**Comment by Bojhan** [comment]
Yup, that sounds exactly like what I would expect!

**Comment by prescottprue** [comment]
Included in [`v0.1.6` release](https://github.com/prescottprue/redux-firebasev3/releases/tag/v0.1.6).
